go/libkb/sig_chain_test.go

```diff
@@ -271,6 +279,30 @@ func doChainTest(t *testing.T, tc TestContext, testCase TestCase) {
 		t.Fatal(fatalStr)
 	}
 
+	if testCase.EldestSeqno != nil && sigchain.EldestSeqno() != *testCase.EldestSeqno {
+		t.Fatalf("wrong eldest seqno: wanted %d but got %d", *testCase.EldestSeqno, sigchain.EldestSeqno())
+	}
+	if testCase.PrevSubchains != nil {
+		if len(testCase.PrevSubchains) != len(sigchain.prevSubchains) {
+			t.Fatalf("wrong number of historical subchains; wanted %d but got %d", len(testCase.PrevSubchains), len(sigchain.prevSubchains))
+		}
+		for i, expected := range testCase.PrevSubchains {
+			received := sigchain.prevSubchains[i]
+			if received.EldestSeqno() != expected.EldestSeqno {
+				t.Fatalf("For historical subchain %d, wrong eldest seqno; wanted %d but got %d", i, expected.EldestSeqno, received.EldestSeqno())
+			}
+			ckf := ComputedKeyFamily{kf: keyFamily, cki: received.GetComputedKeyInfos()}
+			n := len(ckf.GetAllSibkeysUnchecked())
+			if n != expected.Sibkeys {
+				t.Fatalf("For historical subchain %d, wrong number of sibkeys; wanted %d but got %d", i, expected.Sibkeys, n)
+			}
+			m := len(ckf.GetAllSubkeysUnchecked())
+			if m != expected.Subkeys {
+				t.Fatalf("For historical subchain %d, wrong number of subkeys; wanted %d but got %d", i, expected.Sibkeys, m)
+			}
```

Comment:
Would it be easy to throw the current subchain test params and the previous subchain test params into a single list, and then iterate over it once, to avoid the duplication between these new checks and the ones right above? If it's not easy don't worry about it.